docs/ECOS.md

```diff
@@ -205,6 +205,15 @@ Please note, that change of environment variables require restart of the applica
 After preparation done we chroot into /mnt and run cmd from cmdline under specified user and group, output goes to
 /dev/console and is accessible from log of ECO.
 
+## Ability to override default capabilities, mounts and namespaces
+
+By default, we bootstrap OCI configuration using [oci.WithDefaultSpec](https://pkg.go.dev/github.com/containerd/containerd/oci#WithDefaultSpec) function. It fit our expectations for generic ECO we want to run on EVE-OS to be isolated and has limited privileges.
```

Comment:
Do we use this with NOHYPER? If not we should disable that and only allow NOHYPER when service=true.

Reply:
I put few comments about that in doc. Correct, we use WithDefaultSpec if there is no hypervisor available on the system (or if we enforce to run as container using NOHYPER VmMode).